### PR TITLE
CONSOLE-2432: Add i18n externalized strings for events page

### PR DESCRIPTION
**Pull request by jcaianirh**
For: https://issues.redhat.com/browse/CONSOLE-2432

Externalize strings in home/events section

### CODE REVIEW

File: frontend/public/components/events.jsx
Comment (spadgett) [inline frontend/public/components/events.jsx:134]: We should make this one message with parameters

File: frontend/public/components/events.jsx
Comment (spadgett) [inline frontend/public/components/events.jsx:149]: The `Timestamp` should be a parameter to a message using the `Trans` component.

File: frontend/public/components/events.jsx
Comment (spadgett) [inline frontend/public/components/events.jsx:169]: We should be tracking some internal value here instead of saving the translated value in state. This might cause problems if the language is switched.

File: frontend/public/components/events.jsx
Comment (spadgett) [inline frontend/public/components/events.jsx:171]: Sentence case

```suggestion
    all: t('events~All types'),
```

File: frontend/public/components/events.jsx
Comment (spadgett) [inline frontend/public/components/events.jsx:211]:
```suggestion
            title={t('events~All types')}
```

File: frontend/public/components/events.jsx
Comment (spadgett) [inline frontend/public/components/events.jsx:272]: This needs to be a single message instead of concatenating different messages

File: frontend/public/components/events.jsx
Comment (spadgett) [inline frontend/public/components/events.jsx:524]: I think using `{{count}}` in this way won't pluralize correctly. `allCount` needs to be `count` since it's what determines the plural for "events."

File: frontend/public/components/events.jsx
Comment (spadgett) [inline frontend/public/components/events.jsx:542]: `Timestamp` needs to be a parameter to the message.

File: frontend/public/components/events.jsx
Comment (jcaianirh) [reply frontend/public/components/events.jsx:524]: @spadgett, ok, i don't want to use pluralization here...events should always be plural. I can use another variable name aside from count.

File: frontend/public/components/events.jsx
Comment (jcaianirh) [reply frontend/public/components/events.jsx]: added keyword all for all events instead of localized string

**Comment by jcaianirh** [comment]
@spadgett i think i've addressed all review comments here

File: frontend/public/components/events.jsx
Comment (spadgett) [inline frontend/public/components/events.jsx:516]: This shouldn't be localized. Just show `error` as-is.

Comment (jcaianirh) [reply frontend/public/components/events.jsx:516]: @spadgett the error isn't being localized but i'm going to make this more clear.

File: frontend/public/components/events.jsx
Comment (rhamilto) [inline frontend/public/components/events.jsx:308]:
```suggestion
        <title>{title}</title>
```

with `const title = t('events~Events')` added on line 305.

File: frontend/public/components/events.jsx
Comment (rhamilto) [inline frontend/public/components/events.jsx:314]:
```suggestion
        title={title}
```
with additional change above.

**Review by rhamilto — COMMENTED** [review]
Minor DRY nit.

File: frontend/public/components/events.jsx
Comment (jcaianirh) [reply frontend/public/components/events.jsx:308]: thanks @rhamilto

**Comment by jcaianirh** [comment]
@rhamilto updated thanks!

**Review by rhamilto — APPROVED** [review]
/lgtm

/hold for approvals

/assign @ahardin-rh @yapei @sferich888

**Comment by pweil-** [comment]
manually adding `valid-bug` until we work out https://github.com/openshift/release/pull/14440 since the console team is piloting the no feature freeze effort and this PR has all acks, approvals, and lgtms

**Comment by rhamilto** [comment]
/lgtm cancel

Please squash.

**Comment by jcaianirh** [comment]
@rhamilto squashed

**Pull request merged**
